source/server-discovery-and-monitoring/tests/monitoring/replica_set_with_second_seed_removal.yml

```diff
@@ -0,0 +1,100 @@
+description: "Monitoring a topology that is a replica set with second seed removal"
+uri: "mongodb://a,b"
+phases: 
+  # phase 1 - discover topology
+  - 
+    responses: 
+      - 
+        - "a:27017"
+        - 
+          ok: 1
+          ismaster: true
+          setName: "rs"
+          setVersion: 1
+          primary: "a:27017"
+          hosts: 
+            - "a:27017"
+          minWireVersion: 0
+          maxWireVersion: 4
+    outcome: 
+      events: 
+        - 
+          topology_opening_event: 
+            topologyId: "42"
+        - 
+          topology_description_changed_event: 
+            topologyId: "42"
+            previousDescription: 
+              topologyType: "Unknown"
+              servers: []
+            newDescription: 
+              topologyType: "Unknown"
+              servers: 
+                - 
+                  address: "a:27017"
+                  arbiters: []
+                  hosts: []
+                  passives: []
+                  type: "Unknown"
```

Comment:
I expect the second server to still be in the description here - no isMaster response has been processed yet.